app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -102,11 +84,13 @@ protected void onStart() {
             }
         });
 
-        btnLogin.setOnClickListener(v -> {
+        binding.loginContent.btnLogin.setOnClickListener(view -> {
+            if (!validator.validate())
```

Comment:
It was due to the hidden field. I have removed the validation for now. Will move the handling logic in presenter in another PR and then add it